### DISCUSSION TITLE
Update labels based on config not working

### DISCUSSION THREAD

**Discussion by Netail** [Resnovas/smartcloud#165] [Q&A] [answered]
Update a existing label when it does not match the configuration :)

**Comment by TGTGamer** [comment]
How would you do this? Are you meaning as in update the colour and description, which would require me to match the exact same name?

**Reply by Netail** [reply]
After checking if the label with the exact name exists it checks the color and description if they are different from the config -> update. Then add them to the PR/Issue

**Reply by TGTGamer** [reply]
@Netail just confirm for me this isn't already doing that?

https://github.com/Videndum/action-masterminds/blob/c011ff2e03e11a3c40b026255b240fe10e03a5ff/packages/release-mastermind/src/utils/labels.ts#L25-L72

https://github.com/Videndum/action-masterminds/blob/c011ff2e03e11a3c40b026255b240fe10e03a5ff/packages/release-mastermind/src/utils/api/labels.ts#L53-L62

If you are not already having labels updating I think you have actually identified a bug rather than a feature request.

(It's been a while since I worked on this code due to being in my final year at university)

**Reply by Netail** [reply]
Mhm seems like it, it didn't update the color for me...

**Reply by Netail** [reply]
And maybe add a capital check, so update the same if a capital letter changed

**Reply by Netail** [reply]
Color doesn't work at all actually (Except documentation is generated)
![image](https://user-images.githubusercontent.com/11695769/126722261-28c06ccb-e37a-447e-9262-7e8dfc71ae9d.png)
config has the attribute in the following format: `"color": "3ea2e6"`

**Comment by Netail** [comment]
GH Action seems to crash aswell when the default branch is called `main` as it searches for `master`

**Reply by TGTGamer** [reply]
> GH Action seems to crash aswell when the default branch is called `main` as it searches for `master`

Could you confirm which functionality you are trying to use here?

**Reply by Netail** [reply]
PR (same config etc as the created issue)

**Reply by TGTGamer** [reply]
Could you provide the logs for this? The only time which the action would attempt to get the repository is during node.js version checking. Which would imply an error in the `SemVer` versioning.

Does this error still happen if you change to `other` in your versioning type?

**Reply by Netail** [reply]
Wait sorry yes, when testing with the main branch it was on "node"

**Comment by TGTGamer** [comment]
I've got to head to sleep. My brain is hurting from an oncoming migraine - I will pick this up in the morning

**Reply by Netail** [reply]
Np take care of yourself

**Comment by TGTGamer** [comment, marked as answer]
Can I check this has been resolved? 
If so could you mark one of my answers as the answer.

**Reply by Netail** [reply]
resolved
